### PR TITLE
Install latest if no binaries are found

### LINKED ISSUE #183: install latest solc versions by default
Instead of throwing an error as seen here https://github.com/crytic/solc-select/issues/179, we should install the latest solc version and default to it.

### PR DESCRIPTION
**Pull request by 0xicingdeath**
Closes #183

### CODE REVIEW

File: solc_select/__main__.py [hunk 1]
```diff
@@ -84,6 +86,9 @@ def solc_select() -> None:
 
 
 def solc() -> None:
+    if installed_versions() == []:
```
Comment (0xalpharush) [inline solc_select/__main__.py]:
```suggestion
    if not installed_versions():
```

File: solc_select/__main__.py [hunk 2]
```diff
@@ -84,6 +85,9 @@ def solc_select() -> None:
 
 
 def solc() -> None:
+    if not installed_versions():
+        print("No solc versions found, installing latest")
```
Comment (elopez) [inline solc_select/__main__.py]: I wonder if this extra output may confuse tools that expect/consume a specific solc output

Comment (0xicingdeath) [reply solc_select/__main__.py]: We can drop it too; I have no strong preference

Comment (0xicingdeath) [reply solc_select/__main__.py]: Removed in [217103e](https://github.com/crytic/solc-select/pull/203/commits/217103e3d6ef3a3db8dd33409a08c7fa7a4fead2)

Comment (0xalpharush) [reply solc_select/__main__.py]: This also prints https://github.com/crytic/solc-select/blob/217103e3d6ef3a3db8dd33409a08c7fa7a4fead2/solc_select/solc_select.py#L200

**Pull request merged**